R/util.R

```diff
@@ -2920,3 +2920,146 @@ separate_japanese_address <- function(df, address, prefecture_col = "prefecture"
   # call tidyr::unnest_wider so that each element becomes dedicated column like prefecture, city, and street.
   df %>% tidyr::unnest_wider(.exploratory_dummy_column_for_japanese_address, names_repair = ~new_names)
 }
+
+# Constructs a new vector string for the levels.
+# It generates a new character vector based on the base.labels
+# and overrides it with new.labels. The return value has the same
+# length as the base.labels.
+#
+#  new.labels <- c("a", "b", "c")
+#  base.labels <- c("1", "2", "3", "4", "5")
+#  _tam_construct_new_labels (base.labels, new.labels)
+#  [1] "a" "b" "c" "4" "5"
+#
+#  new.labels <- c("a", "b", "c", "d", "e", "f")
+#  base.labels <- c("1", "2", "3", "4", "5")
+#  _tam_construct_new_labels (base.labels, new.labels)
```

Comment:
Function name in the comment here should be updated.

Reply:
Updated function name in the comment.